app/styles/legacy/_musicians-ensembles.scss

```diff
@@ -26,18 +26,18 @@
 }
 
 .playlist-item {
-  padding-left: 50px !important;
   display: flex;
-  flex-direction: column;
-  border-bottom: 0 !important;
-  padding-bottom: 0 !important;
+  border-bottom: 1px dotted $line--light !important;
+  padding-bottom: 24px !important;
```

Comment:
Why all the `!important` flags in this file? I know you didn't start it, but can we get away with not continuing it?

Reply:
yes, but it would require a bunch of extra classes to gain importance over another group of applicable styles.